src/lib/blobs/blobs.ts

```diff
@@ -27,6 +28,7 @@ export type BlobsContext = BlobsContextWithAPIAccess | BlobsContextWithEdgeAcces
 let hasPrintedLocalBlobsNotice = false
 
 export const BLOBS_CONTEXT_VARIABLE = 'NETLIFY_BLOBS_CONTEXT'
+const DEV_REGION = 'dev'
```

Comment:
Yeah, it's a value with no special meaning. We need to have some value for a region, and since we won't actually hit a production service while running `dev` or `serve`, it doesn't really matter what the value is.

I've used `dev` because that's what we're using when invoking functions: https://github.com/netlify/cli/blob/33d2c2c3daa034aff0e95509aaf20a541615e8aa/src/lib/functions/runtimes/js/worker.ts#L31